.eslintrc.cjs

```diff
@@ -0,0 +1,42 @@
+// @ts-check
```

Comment:
It's kind of complicated. You can upgrade to flat configs with eslint 9+ but many plugins including some of those that the repo was using don't support that yet so it would be a larger migration.

I just tried to keep everything as consistent as possible while migrating to a newer version.